app/src/main/java/org/piwigo/ui/settings/SettingsViewModel.java

```diff
@@ -38,13 +39,19 @@ public SettingsViewModel(UserManager userManager, UserRepository userRepository,
         this.userRepository = userRepository;
         this.resources = resources;
         this.userManager = userManager;
+        this.account = userManager.getActiveAccount().getValue();
 
     }
 
     public void onLogoutClick(){
+        if(account != null){
         userRepository.logout(userManager.getActiveAccount().getValue())
                 .compose(applySchedulers())
                 .subscribe(new LogoutSubscriber());
+        } else {
+          Throwable e =  new Throwable("Empty account");
+            logoutError.setValue(e);
```

Comment:
Is this really a problematic use case? If we are already logged out and someone logs out another time, why doing something at all in this else branch?

On the other side I think it would be best to directly show either the LoginActivity or the ManageAccountAcitivy directly after logout, because we want the use to select which account to use if there are still accounts, and to add a new one if the last account was deleted.